.github/workflows/tdd-verification.yml

```
@@ -0,0 +1,252 @@
+          import os
+          import re
+          from pathlib import Path
+
+          test_files = list(Path('tests').glob('**/*.py'))
+          violations = []
+
+          for test_file in test_files:
+              if test_file.name.startswith('test_') and not test_file.name.startswith('conftest'):
+                  with open(test_file, 'r') as f:
+                      content = f.read()
+                      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)